Content.Client/Humanoid/HumanoidVisualizerSystem.cs

```diff
@@ -274,6 +274,18 @@ private void SetSpriteVisibility(EntityUid uid, HashSet<HumanoidVisualLayers> la
         visible &= humanoid.BaseLayers.TryGetValue(markingPrototype.BodyPart, out var setting)
            && setting.AllowsMarkings;
 
+        // Marking coloring
+        
+        var skinColor = humanoid.SkinColor;
+        if (visible && setting != null) skinColor.A = setting.LayerAlpha;
+        List<Color> marking_colors = MarkingColoring.GetMarkingLayerColors(
```

Comment:
any variable defined with its type prefacing the name can just have the type replaced with `var` (see: most variable declarations in this codebase)

Reply:
the only time you'd not want to do that is, of course, if a variable needs to be declared before it's initialized with a value